packages/main/src/sync.ts

```diff
@@ -31,17 +31,17 @@ export async function sync(accountId: cache.ID, manager: ConnectionManager) {
       .request(request.actions.getBox(specifier))
       .toPromise()
     const boxId = cache.persistBoxState(accountId, box)
-    await new BoxSync({
-      accountId,
-      box,
-      boxId,
-      manager
-    }).sync()
+    // await new BoxSync({
+    //   accountId,
+    //   box,
+    //   boxId,
+    //   manager
+    // }).sync()
   }
   const contactApiClient = AccountManager.getContactsApiClient(
     String(accountId)
   )
-  contactApiClient && contactApiClient.downloadContacts(accountId)
+  contactApiClient && (await contactApiClient.downloadContacts(accountId))
```

Comment:
Good idea to put the `await` here. For one thing if there is an error in the `downloadContacts` call the error will now propagate to the result of `sync`. Without this `await` the error would be lost.

Later we can talk about how to run asynchronous calls in parallel.